core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXV1Endpoint.java

```diff
@@ -359,4 +362,16 @@ void setClusterMaintenanceModeByCrn(@PathParam("crn") String crn,
     @ApiOperation(value = RENEW_CERTIFICATE, produces = MediaType.APPLICATION_JSON, notes = Notes.RENEW_CERTIFICATE_NOTES,
             nickname = "renewDistroXCertificate")
     FlowIdentifier renewCertificate(@PathParam("crn") String crn);
+
+    @PUT
+    @Path("name/{name}/rotate_certificates")
```

Comment:
Should we call it rotate_autotls_certificates similarly to the permissions?

Reply:
it's coming from `StackV4Endpoint`. I can change this, but then I think I would change in `StackV4Endpoint` too to keep them in sync. Shall I change them?

Reply:
yes, please.